session/session.go

```diff
@@ -962,12 +962,12 @@ func (s *session) ParseSQL(ctx context.Context, sql, charset, collation string)
 }
 
 func (s *session) SetProcessInfo(sql string, t time.Time, command byte, maxExecutionTime uint64) {
-	var db interface{}
+	var db string
```

Comment:
Remove the follow 9 lines, you can simply use s.sessionVars.CurrentDB and sql inside the pi assignment.